eduNEXT/xblock-filesmanager
# fix: add date and file size information

### Description

This PR allows to showing date and file size information.

### Testing instructions

1. Upload a file
2. Verify that Chonky shows the right information for file size and date.

![image](https://github.com/eduNEXT/xblock-filesmanager/assets/33465240/5c351a4c-b8bb-49fb-bb03-8b3bf1369b29)

### Concerns

The date is in UTC format and needs to be localized in the frontend. Any ideas teams?

**Merge checklist:**
Check off if complete *or* not applicable:
